dell/src/test/java/org/apache/iceberg/dell/ecs/TestEcsSeekableInputStream.java

```diff
@@ -27,6 +27,8 @@
 import org.junit.ClassRule;
 import org.junit.Test;
```

Comment:
this is still using Junit4 tests and hasn't been moved to Junit5

Reply:
Same as above, this one too uses ClassRule.